lib/class-wp-block-patterns-registry.php

```diff
@@ -42,6 +42,18 @@ public function register( $pattern_name, $pattern_properties ) {
 			return false;
 		}
 
+		if ( ! isset( $pattern_properties['title'] ) || ! is_string( $pattern_properties['title'] ) ) {
+			$message = __( 'Pattern title must be a string.', 'gutenberg' );
+			_doing_it_wrong( __METHOD__, $message, '8.5.0' );
+			return false;
+		}
+
+		if ( ! isset( $pattern_properties['content'] ) || ! is_string( $pattern_properties['content'] ) ) {
+			$message = __( 'Pattern content must be a string.', 'gutenberg' );
+			_doing_it_wrong( __METHOD__, $message, '8.5.0' );
+			return false;
+		}
```

Comment:
The change here needs dedicated Core patch.

Reply:
I am not sure where to upload it :) https://core.trac.wordpress.org/attachment/ticket/50550/50550-doing-it-wrong.patch